webapp/src/components/ActivityPage/Transaction/Status/Status.js

```diff
@@ -16,13 +16,23 @@ export default class Status extends React.PureComponent {
     tx: transactionType
   }
 
+  mounted = false
```

Comment:
Just in case.

We opted out of this syntax in the past to keep consistent with the rest of the Components. You may you want to move this to the `constructor`. I'm ok with it being here tbh.